hudi-common/src/test/java/org/apache/hudi/avro/TestMercifulJsonConverter.java

```diff
@@ -73,11 +68,7 @@ public void basicConversion() throws IOException {
   }
 
   @ParameterizedTest
-  @ValueSource(strings = {
-      "{\"first\":\"John\",\"last\":\"Smith\"}",
-      "[{\"first\":\"John\",\"last\":\"Smith\"}]",
-      "{\"first\":\"John\",\"last\":\"Smith\",\"suffix\":3}",
-  })
+  @MethodSource("dataNestedJsonAsString")
```

Comment:
Have converted this MethodSource and moved it Test Base class to be reused by MercifulJsonToRowConverter